### PR TITLE
Fix enabling Help item in context menu for Recognition node

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Currently the `Help..` item is disabled in the context menu when a Recognition node is selected.

All the other nodes without WREN objects have the same issue: ContactProperties, Focus, Zoom, etc.

### CODE REVIEW

File: docs/reference/changelog-r2020.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Released on XXX YYY, 2020.
     - Fixed [Battery](robot.md#wb_robot_battery_sensor_enable) sensor when the [Robot](robot.md) contains a [Propeller](propeller.md) device ([#1801](https://github.com/cyberbotics/webots/pull/1801)).
     - Fixed [Motor](motor.md) target position not updated when changing the `position` field of a [JointParameters](jointparameters.md) node ([#2326](https://github.com/cyberbotics/webots/pull/2326)).
     - Fixed [`wb_motor_get_target_position`](motor.md#wb_motor_get_target_position) function if the joint initial position is not 0 ([#2326](https://github.com/cyberbotics/webots/pull/2326)).
+    - Fixed disabled `Help..` item in context menu for some nodes ([#2327](https://github.com/cyberbotics/webots/pull/2327)).
```
Comment (omichel) [inline docs/reference/changelog-r2020.md:23]:
```suggestion
    - Fixed the disabled `Help...` item in the context menu for some nodes ([#2327](https://github.com/cyberbotics/webots/pull/2327)).
```

**Review by omichel — CHANGES_REQUESTED** [review]
Thank you.

File: src/webots/scene_tree/WbSceneTree.cpp [hunk 2]
```diff
@@ -1119,8 +1119,9 @@ void WbSceneTree::updateSelection() {
       // baseNode = NULL if none or multiple instances exists
       baseNode = baseNode->getSingleFinalizedProtoInstance();
 
-    if (baseNode && !baseNode->areWrenObjectsInitialized())
+    if (baseNode && !(baseNode->areWrenObjectsInitialized() || baseNode->isPostFinalizedCalled()))
```
Comment (DavidMansolino) [inline src/webots/scene_tree/WbSceneTree.cpp:1122]: Not sure about the implication, but wouldn't it be possible to keep only the ``isPostFinalizedCalled`` conditionand remove the ``areWrenObjectsInitialized``.

Comment (stefaniapedrazzi) [reply src/webots/scene_tree/WbSceneTree.cpp:1122]: Yes, I think there won't be any difference. I will check.

**Review by omichel — APPROVED** [review]
Thank you.

**Comment by stefaniapedrazzi** [comment]
I just fixed some conflicts in the ChangeLog.

**Review by omichel — APPROVED** [review]

**Pull request merged**